Rakefile

```diff
@@ -62,7 +66,9 @@ def sed(filename, replacements)
 end
 
 def new_version
-  parts = DropzonejsRails::VERSION.split('.')
-  parts[2] = parts[2].to_i + 1
-  parts.join '.'
+  DropzonejsRails::VERSION.next
```

Comment:
Although more classy and concise, I don't think that honors the same behavior:

```ruby
'1.1.1'.next
=> "1.1.2"
'1.1.2'.next
# => "1.1.3" # This one's OK
'1.1.9'.next
# => "1.2.0" # This would bump a minor version number instead of bumping to '1.1.10'...
'1.9.9'.next
# => "2.0.0" # And this would even release a new major version instead of bumping to '1.9.10'.
```

This breaks SemVer a bit. Anyway - leaving that aside - I'm open to changing the versioning scheme to match Dropzone's version. That would really ease things, wouldn't it? What do you think?